src/app/issues-viewer/card-view/card-view.component.html

```diff
@@ -18,12 +18,14 @@
     </mat-card>
   </div>
 
-  <div *ngFor="let issue of this.issues$ | async; index as i">
-    <app-issue-pr-card [issue]="issue" [dropdownFilter]="issues.dropdownFilter"></app-issue-pr-card>
-  </div>
+  <mat-card-content style="position: relative;">
```

Comment:
mat-card should enclose mat-card-content

Reply:
Since we are not using mat-card for this, I think div is a better element to use

Reply:
You're right, I'll work on tidying it up, thanks!